python/cudf/cudf/_fuzz_testing/avro.py

```diff
@@ -105,12 +105,16 @@ def set_rand_params(self, params):
                 if param == "columns":
                     col_size = self._rand(len(self._df.columns))
                     params_dict[param] = list(
-                        np.unique(np.random.choice(self._df.columns, col_size))
+                        np.unique(
+                            np.random.default_rng(2).choice(
```

Comment:
It seems like in lots of these cases, we should be seeding an RNG once at the beginning of the function and then using the same RNG instance (so that subsequent calls have new results rather than reseeding with the same value).